plugins/filter_apm_kubernetes_labels/filter_kubernetes_labels.c

```diff
@@ -97,12 +98,34 @@ static int configure(struct kubernetes_labels_ctx *ctx, struct flb_filter_instan
     else
         ctx -> projname_labe1 = DEFAULT_PROJECTNAME_LABEL;
 
+    char* proj_name = getenv(SFAPM_PROJECT_NAME);
+    if(proj_name)
+        ctx -> projname = proj_name;
+    else
+        ctx -> projname = DEFAULT_PROJECTNAME;
+
 
     char* app_name_label = getenv(SFAPM_APPNAME_LABEL);
     if(app_name_label)
         ctx -> appname_labe1 = app_name_label;
     else
         ctx -> appname_labe1 = DEFAULT_APPNAME_LABEL;
+
+    char* app_name = getenv(SFAPM_APP_NAME);
+    if(app_name)
+        ctx -> appname = app_name;
+    else
+        ctx -> appname = DEFAULT_APPNAME;
+    
+    bool monitor_pods_label = getenv(MONITOR_ALL_PODS);
```

Comment:
change monitor_pods_label variable name, this doesn't convey anything related to MONITOR_ALL_PODS flag